packages/components/src/SubmitChallengeModal.tsx

```diff
@@ -199,8 +199,8 @@ export class SubmitChallengeModal extends React.Component<
         <div>
           <ModalSectionHeader>Deposit CVL tokens for your challenge</ModalSectionHeader>
           <CopyLarge>
-            This challenge requires a deposit of 1,000 CVL tokens. If you don’t have enough CVL tokens,{" "}
-            <a href="#">buy more here</a>.
+            This challenge requires a deposit of {this.props.minDeposit} CVL tokens. If you don’t have enough CVL
```

Comment:
run this value through the formatting function?

Reply:
The formatting function gets run in the dapp and the prop is passed as a formatted string